editor-components/modal/notificationmodal2/notificationmodal2.module.scss

```diff
@@ -0,0 +1,112 @@
+    width: 100%;
+
+    .max-content {
+      width: 420px;
+      height: 300px;
+      border-radius: 30px;
+      background-color: var(--composer-html-background);
+      margin: auto;
+      padding: 40px;
+
+      .content {
+        position: relative;
+        margin: auto;       
+        flex-direction: column;
+        justify-content: center;
+        align-items: center;
+        display: flex;
+        min-height: 100%;
+        gap: 20px;
+
+        .icon {
+          color: var(--composer-font-color-primary);
+          margin: 0;
+          font-size: 60px;
+          border-radius: 20px;
+          display: flex;
+          align-items: center;
+          justify-content: center; 
+        }
+  
+        .notification-title {
+          margin: 0;
+          text-align: center;
+          align-items: center;
+          font-size: 20px;
+          font-family: sans-serif;
+          display: flex;
+          color: var(--composer-font-color-primary);
+        }
+  
+        .displayer-message {
+          margin: 0;
+          width: 100%;
+          height: fit-content;
+          font-size: 16px;
+          font-family: sans-serif;
+          color: gray;
+          text-align: center;
+          word-wrap: break-word;
+          align-items: center;
+          justify-content: center;
+          color: var(--composer-font-color-primary);
+        }
+  
+        .send-mail {
+          margin: 0;
+          transition: 1s;
+          display: flex;
+          width: 100%;
+          justify-content: center;
+          align-items: center;
+          width: 100%;
+          display: flex;
+          justify-content: center;
+  
+          .input-group {
+            position: relative;
+            width: 360px;
+
+            .input {
+              width: 100%;
+              height: 45px;
+              padding: 16px 10px;
+              padding-right: 140px; 
+              border-radius: 20px;
+              border: 0.5px solid rgba(var(--composer-font-color-primary-rgb), 0.3);
+              background: var(--composer-html-background);
+              color: var(--composer-font-color-primary);
+
+
+              &:focus {
+                outline-color: var(--composer-secondary-color);
+              }
+            }
+  
+            .button {
+              position: absolute;
+              right: 5px;
+              top: 50%;
+              transform: translateY(-50%);
+              width: 100px;
+              height: calc(85%);
+              border-radius: 20px;
+              background-color: var(--composer-secondary-color);
+              border: none;
+              box-shadow: var(--composer-box-shadow);
+              color: white;
+              cursor: pointer;
+              font-size: 14px;
+
+              &:hover {
+                color: white;
+                background-color: var(--composer-secondary-color);
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+  
```

Comment:
Add media queries.
For screens that their width is under the modal's width, make their width 100%.